src/app/components/snapshots/list-of-snapshots/list-of-snaphshots.component.spec.ts

```diff
@@ -32,6 +33,9 @@ export class MockedActivatedRoute {
                     get(name: string): string {
                         return '1';
                     }
+                },
+                data: {
```

Comment:
@ziajka We shouldn't add this manually but we still don't know why line#77 doesn't work

Reply:
Let's add to the items we need to walk-through.